debian/base/Dockerfile

```diff
@@ -57,7 +60,7 @@ RUN echo "===> Updating debian ....." \
                 netcat \
                 python=${PYTHON_VERSION} \
```

Comment:
Should we add support for https://... repos too ? The staging repos are https://...

https://github.com/confluentinc/cp-docker-images/commit/ffdf60c6e7cf5232a822347b351b194fd4a0a9af#diff-8a469355c509e07885a8934541d1dc03L60